platform/sun50i/devices.c

```diff
@@ -36,6 +39,15 @@ static struct device r_twd    __device;
 static struct device sy8106a __device;
 #endif
 
+#if CONFIG_PMIC_AXP803
+static struct device axp803_pmic = {
+	.name = "axp803-pmic",
+	.drv  = &axp803_pmic_driver.drv,
+	.bus  = &r_i2c,
+	.addr = 0x34,
```

Comment:
There's a macro for this

Reply:
For `.addr`? I am not sure what you mean - the `axp803_regulator` has the address `0x34` hard-coded as well.

Reply:
Hmm, well it's there https://github.com/crust-firmware/crust/blob/master/include/drivers/regulator/axp803.h#L14, looks like I forgot to use it in `devices.c`

Reply:
I'll add that in a separate commit and include the macro in the PMIC driver as well.

Reply:
> I'll add that in a separate commit and include the macro in the PMIC driver as well.

Done.